pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -83,6 +84,32 @@ func findPlatformProvider(in *executor.Input) (name string, cfg *config.Platform
 	return
 }
 
+func LoadVpcConfiguration(in *executor.Input, vpcConfiguration string, ds *deploysource.DeploySource) (types.AwsVpcConfiguration, bool) {
```

Comment:
Is this fine to define as a private function?
```suggestion
func loadVpcConfiguration(in *executor.Input, vpcConfiguration string, ds *deploysource.DeploySource) (types.AwsVpcConfiguration, bool) {
```